### PR TITLE
Demand progress disclosure

### PR DESCRIPTION
**Pull request by Mbalcorta**
Add form for to submit to actionKit and add demand progress disclosure

### CODE REVIEW

File: breaktheinternet/index.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
     <link rel="stylesheet" href="css/style.css?v=CACHE_BUST">
 
     <script src="js/jquery.min.js"></script>
+    <script type="text/javascript" src="//act.demandprogress.org/resources/actionkit.js"></script>
     <script src="js/app.js?v=CACHE_BUST"></script>
```
Comment (tibetsprague) [inline breaktheinternet/index.html]: you should update the CACHE_BUST  to something new (the current timestamp is good) to make sure it gets updated in peoples browsers

File: breaktheinternet/index.html [hunk 2]
```diff
@@ -51,38 +52,78 @@ <h1>
...
-        <div class="social">
-          <a class="facebook" href="https://www.facebook.com/sharer.php?u=https%3A%2F%2Fbattleforthenet.com%2Fbreaktheinternet" target="_blank">Share on Facebook</a>
-          <a class="twitter" href="https://twitter.com/intent/tweet?related=fightfortheftr&text=The%20FCC%20is%20days%20away%20from%20voting%20to%20kill%20NetNeutrality%2C%20but%20Congress%20can%20still%20stop%20them.%20On%20December%2012th%20we'll%20%23BreakTheInternet%20to%20stop%20censorship%2C%20throttling%2C%20and%20extra%20fees%3A%20https%3A%2F%2Fbattleforthenet.com%2Fbreaktheinternet" target="_blank">Share on Twitter</a>
-        </div>
-      </section>
+        
+            <div class="ak-pagetype-signup" id="dp_form" style="display:none">
```
Comment (tibetsprague) [inline breaktheinternet/index.html]: the indentation looks a bit off here

File: breaktheinternet/index.html (hunk 1 above)
Comment (Mbalcorta) [reply breaktheinternet/index.html]: Would it then be changed to this? js/app.js?v=1512686348638

**Comment by tibetsprague** [comment]
looks good, only thing is the pull request should be to merge into gh-pages not master I think. or maybe one for both? that sounds good

**Comment by tibetsprague** [comment]
oh also, I just wanted to make sure this was fully tested, and is only sending leads to DP's AK when org=dp

**Pull request merged**